controllers/v1/observationSubmissionsController.js

```diff
@@ -205,8 +224,8 @@ module.exports = class ObservationSubmissions extends Abstract {
           entityInformation: entityDocument.metaInformation,
           solutionId: solutionDocument._id,
```

Comment:
@borkarsaish65 solution is a mandatory thing? are we only making program optional?

Reply:
task mentioned that programs need to be optional @VISHNUDAS-tunerlabs